.github/workflows/main-deploy-docs.yml

```diff
@@ -21,5 +21,6 @@ jobs:
       - name: Docs
         env:
           DOCS_TOKEN: ${{ secrets.DOCS_TOKEN }}
+          TIBANNA_VERSION: 2.2.0
```

Comment:
This version won't matter for the docs build though right?

Reply:
I think the version matters. It is also used in the `conf.py` in the docs folder. 
![image](https://user-images.githubusercontent.com/53857412/192358934-0e59fa81-b26c-40ea-ba90-f47c67d9b39a.png)